docs/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * New vertices() method on MeshCreator trait. ([#946])
 * Support for text alignment (align left, center, right). ([#965])
 * Support for multiline text. ([#965])
+* Added custom aspect ratio to OrthoCamera. ([#983])
 
```

Comment:
You need to also link the pull request like so:
[#983]: https://github.com/amethyst/amethyst/pull/983/

Reply:
:+1: